common/app/common/akka.scala

```diff
@@ -3,31 +3,38 @@ package common
 import akka.agent.Agent
 import play.api.libs.concurrent.{Akka => PlayAkka}
 import scala.concurrent.duration._
-import play.api.Play
+import play.api.{Mode, Environment, Play}
 import scala.concurrent.ExecutionContext
 import akka.actor.ActorSystem
-import play.api.Play.current
 
 object ExecutionContexts extends ExecutionContexts
 
 trait ExecutionContexts {
   implicit lazy val executionContext: ExecutionContext = play.api.libs.concurrent.Execution.Implicits.defaultContext
-  lazy val actorSystem: ActorSystem = PlayAkka.system
-  lazy val feedsRecorderExecutionContext: ExecutionContext = PlayAkka.system.dispatchers.lookup("akka.actor.feed-recorder")
+  lazy val actorSystem: ActorSystem = PlayAkka.system(Play.current)
+  lazy val feedsRecorderExecutionContext: ExecutionContext = PlayAkka.system(Play.current).dispatchers.lookup("akka.actor.feed-recorder")
 }
 
 object AkkaAgent extends ExecutionContexts {
   def apply[T](value: T): Agent[T] = Agent(value)
 }
 
-object AkkaAsync extends ExecutionContexts {
+class AkkaAsync(env: Environment, actorSystem: ActorSystem) {
+  implicit val ec: ExecutionContext = actorSystem.dispatcher
 
   def apply(body: => Unit): Unit = after(1.second){ body }
```

Comment:
that's great for readability, scala can really let you write confusing code otherwise - I'd probably have made the old "apply" call the new after1s method, and left a comment to say it's going away, but that's because I might forgot 😛